packages/superset-ui-legacy-preset-chart-nvd3/src/LineMulti/LineMulti.jsx

```diff
@@ -31,8 +31,10 @@ const propTypes = {
   annotationData: PropTypes.object,
   datasource: PropTypes.object,
   formData: PropTypes.object,
-  onAddFilter: PropTypes.func,
-  onError: PropTypes.func,
+  hooks: PropTypes.shape({
+    onAddFilter: PropTypes.func,
+    onError: PropTypes.func,
+  }),
 };
 const defaultProps = {
   onAddFilter() {},
```

Comment:
You don't need this.now.

Reply:
oh, didn't see that there. Wonder why is was failing on master if it has that fallback. 🤷‍♂

Reply:
There is an ongoing issue with build tools, probably not related to this.